replica/table.cc

```diff
@@ -1503,7 +1503,7 @@ future<> compaction_group::stop() noexcept {
 }
 
 void compaction_group::clear_sstables() {
-    _main_sstables = _t._compaction_strategy.make_sstable_set(_t._schema);
+    *_main_sstables = _t._compaction_strategy.make_sstable_set(_t._schema);
     _maintenance_sstables = _t.make_maintenance_sstable_set();
```

Comment:
very subtle, either this or wrap make_lw_shared around creation of a new set. I am fine with either.

Reply:
thanks for working on this

Reply:
ahh, right! changed accordingly. now that we're here, there is no reason to make it more correct.